src/commands/compile/spec/loadDependencies.spec.ts

```diff
@@ -140,12 +142,14 @@ describe('loadDependencies', () => {
   })
 
   test('it should load dependencies for a job <h4> Dependencies </h4>', async (done) => {
-    spyOn(internalModule, 'getJobInit').and.returnValue(
-      `\n* JobInit start;\n${fakeServiceInit}\n* JobInit end;`
-    )
-    spyOn(internalModule, 'getJobTerm').and.returnValue(
-      `\n* JobTerm start;\n${fakeServiceTerm}\n* JobTerm end;`
-    )
+    spyOn(internalModule, 'getJobInit').and.returnValue({
+      content: `\n* JobInit start;\n${fakeServiceInit}\n* JobInit end;`,
```

Comment:
the description says Job dependencies but the test says "ServiceInit" - if we're re-using the serviceInit in the test for the Job, perhaps we should add a comment to explain the discrepancy in the variable naming?

Reply:
upda variable naming.